hermes/probe/delete/delete.go

```diff
@@ -89,55 +73,55 @@ func DeleteFile(ctx context.Context, fileID int32, target *probe.Target, client
 
 	start := time.Now()
 	if err := file.Delete(ctx); err != nil {
-		var status m.ExitStatus
+		var status metrics.ExitStatus
 		switch err {
 		case storage.ErrObjectNotExist:
-			status = m.FileMissing
+			status = metrics.FileMissing
 		case storage.ErrBucketNotExist:
-			status = m.BucketMissing
+			status = metrics.BucketMissing
 		default:
-			status = m.ProbeFailed
+			status = metrics.ProbeFailed
 		}
 
-		target.LatencyMetrics.APICallLatency[m.APIDeleteFile][status].Metric("hermes_api_latency_s").AddFloat64(time.Now().Sub(start).Seconds())
+		target.LatencyMetrics.APICallLatency[metrics.APIDeleteFile][status].Metric("hermes_api_latency_seconds").AddFloat64(time.Now().Sub(start).Seconds())
```

Comment:
"hermes_api_latency_seconds" is repeated multiple times. It should be a string constant.

Reply:
I've refactored this into a constant and used it throughout.

I've also added a TODO to add this into metrics.go